docs/3.0/search.md

```diff
@@ -252,3 +252,50 @@ class Avo::Resources::Order < Avo::BaseResource
   }
 end
 ```
+
+## Searching within associations
+
+In some cases, you might need to search for records based on attributes of associated models. This can be achieved using the `ransack` gem along with ActiveRecord associations. Here's an example of how to perform such a search within associations:
```

Comment:
```suggestion
In some cases, you might need to search for records based on attributes of associated models. This can be achieved by adding a few things to the search query. Here's an example of how to do that:
```